Source/Blazorise/Components/Table/TableRow.razor

```diff
@@ -2,6 +2,8 @@
 @inherits BaseDraggableComponent
 <tr @ref="@ElementRef" id="@ElementId" class="@ClassNames" style="@StyleNames" @onclick="@OnClickHandler"
     draggable="@DraggableString"
+    @onmouseover="@(MouseOver.HasDelegate ? OnMouseOverHandler : null)"
```

Comment:
Do we want to add `@onmouseover:stopPropagation` here?

Reply:
Do we? I don't know. What about be the benefit?

Reply:
It can prevent mouse events from bubbling up to the table and possibly slowing down the grid.

Reply:
It's literally a row, I'm not sure how much further it will even bubble up, and it won't be slowing down since there won't be interops which are the actual thing that can slow down blazor.